docs/guide/from-2022a-to-2022b.md

```diff
@@ -0,0 +1,119 @@
+
+### How Are EXTERNPROTO (and IMPORTABLE EXTERNPROTO) Declared
+
+When opening a world file using a text editor, declared PROTO will look like:
+
+```
+#VRML_SIM R2022b utf8
+
+EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2022b/projects/objects/backgrounds/protos/TexturedBackground.proto"
+IMPORTABLE EXTERNPROTO "../protos/RectangleArena.proto"
+
+WorldInfo {
+  basicTimeStep 16
+}
+Viewpoint {
+}
+TexturedBackground {
+}
+```
+
+In the world above, the PROTO **TexturedBackground** will be retrieved from the corresponding URL.
+As shown by the snippet no instances of **RectangleArena** are present at load time however by flagging it as `IMPORTABLE EXTERNPROTO` we denote that this object may appear during the execution.
+Similarly, a PROTO may reference another PROTO, and in this case a declaration is also necessary.
+For example the following **MyBox** PROTO uses the **Plastic** appearance and therefore needs to declare it.
+
+```
+#VRML_SIM R2022b utf8
+
+EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2022b/projects/appearances/protos/Plastic.proto"
+
+PROTO MyBox [
+  field SFVec3f  size    0.1 0.1 0.1  # Defines the size of the box.
+]
+{
+  Shape {
+    appearance Plastic {
+    }
+    geometry Box {
+      size IS size
+    }
+  }
+}
+
+```
+
+In general, the URL can be defined multiple ways:
+
+```
+# remote PROTO
+EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2022b/projects/objects/backgrounds/protos/TexturedBackground.proto"
+
+# local PROTO (absolute path)
+EXTERNPROTO "/home/user/Documents/my_project/protos/TexturedBackground.proto"
+
+# local PROTO (relative path, be it relative to a world or to the PROTO itself depending on the situation)
+EXTERNPROTO "../protos/TexturedBackground.proto"
+```
+The location (URL) for official PROTO can be found in the file: `WEBOTS_HOME/resources/proto-list.xml`.
+When inserting PROTO nodes using the **add-node** button, a declaration is added automatically.
+
+### IMPORTABLE EXTERNPROTO Declaration
+
+PROTO nodes that are spawned by a supervisor and therefore may appear later in the execution need to be declared in the world file in the same manner, using however the key word `IMPORTABLE EXTERNPROTO` rather than just `EXTERNPROTO`.
+Note that declaring a node as `IMPORTABLE EXTERNPROTO` automatically implies that the PROTO is also an `EXTERNPROTO`.
+In other words, only one declaration suffices.
+
+The list of PROTO nodes that can be spawned using a Supervisor is shown in the `IMPORTABLE EXTERNPROTO` panel, which can be opened by clicking the similarly named button:
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/44834743/179792050-999986e9-25ef-4318-9da1-bd5739e245b8.png">
+</p>
+
+Doing so will open the panel:
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/44834743/179791656-9e2bf259-cd15-4fa3-922d-949f9db420b1.png">
+</p>
```

Comment:
Usually the images are centered vertically in the page, but this is aligned on the left.